src/braket/aws/aws_quantum_task.py

```diff
@@ -130,9 +133,12 @@ def __init__(
             aws_session (AwsSession, optional): The `AwsSession` for connecting to AWS services.
                 Default is `None`, in which case an `AwsSession` object will be created with the
                 region of the task.
-            poll_timeout_seconds (int): The polling timeout for result(), default is 120 seconds.
-            poll_interval_seconds (int): The polling interval for result(), default is 0.25
+            poll_timeout_seconds (float): The polling timeout for result(), default is 120 seconds.
+            poll_interval_seconds (float): The polling interval for result(), default is 0.25
```

Comment:
Wait, you mean 0.25 _isn't_ an `int`? :open_mouth:

Reply:
Mind blown, right?